team-props/git-hooks.gradle

```diff
@@ -0,0 +1,37 @@
+static def isLinuxOrMacOs() {
+    def osName = System.getProperty('os.name').toLowerCase(Locale.ROOT)
+    return osName.contains('linux') || osName.contains('mac os') || osName.contains('macos')
+}
+
+task copyGitHooks(type: Copy) {
+    description 'Installs the git hooks from team-props/git-hooks.'
+    from("${rootDir}/team-props/git-hooks/") {
+        include '**/*.sh'
+        rename '(.*).sh', '$1'
+    }
+    into "${rootDir}/.git/hooks"
+    onlyIf {
+        return isLinuxOrMacOs()
+    }
+}
+
+task installGitHooks(type: Exec) {
+    description 'Copies the resource directory to the target directory.'
```

Comment:
I'm a bit confused: this task is named install and copies, the other task is named copy and installs ❓

Reply:
I forgot to change the description after renaming and refactoring :D

Reply:
cc23245